### PR TITLE
Update submodules.json

### PR DESCRIPTION
**Pull request by sparsh-kamat**

### CODE REVIEW

File: submodules.json [hunk 1]
```diff
@@ -8,5 +8,10 @@
         "name" : "node-boilerplate",
         "url" : "https://github.com/hagopj13/node-express-boilerplate",
         "tags" : ["node", "nodejs"]
+    },
+     {
```
Comment (mintbomb27) [inline submodules.json:12]: Please lint the JSON properly 😅. Would be great if you could remove that whitespace.

File: submodules.json [hunk 2]
```diff
@@ -8,5 +8,10 @@
...
+        "name" : "jquery-boilerplate",
+        "url" : "https://github.com/jquery-boilerplate/jquery-boilerplate",
+        "tags" : ["jquery", "jqueryjs"]
```
Comment (mintbomb27) [inline submodules.json:15]: Could add a `js` tag too here!

**Review by mintbomb27 — CHANGES_REQUESTED** [review]
Thanks for the contribution! Great repo! Please check the comments :)

**Comment by sparsh-kamat** [comment]
Hi, I've done the changes requested.

**Review by mintbomb27 — APPROVED** [review]
Thanks a lot for the contribution! LGTM!

**Pull request merged**